### PR TITLE
add fix proj

### PR DESCRIPTION
**Pull request by dmorozko**
new project with Dockerfile

### CODE REVIEW

File: backend_module/Docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
```
Comment (kurenchuksergey) [inline backend_module/Docker-compose.yml:2]: add also database

File: backend_module/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+FROM openjdk:11
```
Comment (kurenchuksergey) [inline backend_module/Dockerfile:12]: i already send you example, how build this image without jdk on local machine ?

File: backend_module/Docker-compose.yml [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+    hostname: localhost
+    ports:
+      - 5432:5432
+    environment:
+      MYSQL_ROOT_PASSWORD: rootpass
+      MYSQL_USER: testuser
+      MYSQL_PASSWORD: testpass
+      MYSQL_DB: teste
+  server:
+    image: spring-mysql
```
Comment (kurenchuksergey) [inline backend_module/Docker-compose.yml:15]: pls register your docker file in docker hub

Comment (dmorozko) [reply backend_module/Docker-compose.yml:15]: https://hub.docker.com/repository/docker/morozzyaka/spring-mysql

**Pull request merged**